app/Exceptions/StudentHasAlreadyGroupInCourseException.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Exceptions;
+
+use Exception;
+
+class StudentHasAlreadyGroupInCourseException extends Exception
+{
+    //
```

Comment:
Missing the exception constructor with a default message.
Please fix the other exceptions too.